rpc/tests/utils/client.rs

```diff
@@ -54,6 +57,11 @@ pub async fn new_test_client(auth_level: AuthLevel) -> Result<Client> {
     Ok(client)
 }
 
+pub async fn new_test_client(auth_level: AuthLevel) -> Result<Client> {
+    let url = env_or("CELESTIA_RPC_URL", CELESTIA_RPC_URL);
+    new_test_client_with_url(auth_level, &url).await
```

Comment:
sorry i wasn't thinking straight when i said `env_or` is needed.
your previous code was correct.

```suggestion
    new_test_client_with_url(auth_level, CELESTIA_RPC_URL).await
```

Reply:
Oh, okay!

Pushed it now.